### DISCUSSION TITLE
时不时就出现这个错误，然后就蓝屏了，然后在服务器重启docker又好了。

### DISCUSSION THREAD

**Discussion by sbkmlvb** [steedos/steedos-platform#3897] [Q&A]
steedos_1             | This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). The promise rejected with the reason:
steedos_1             | TypeError: Cannot read property 'admin_menus' of undefined
steedos_1             |     at /app/node_modules/@steedos/objectql/src/types/app.ts:243:47
steedos_1             |     at step (/app/node_modules/tslib/tslib.js:144:27)
steedos_1             |     at Object.next (/app/node_modules/tslib/tslib.js:125:57)
steedos_1             |     at fulfilled (/app/node_modules/tslib/tslib.js:115:62)
steedos_1             |     at /app/node_modules/meteor-promise/fiber_pool.js:43:39
![微信截图_20221031210225](https://user-images.githubusercontent.com/54029752/199014098-1b61d8fc-c1b2-4e3a-96e7-896f5b10b942.png)

**Comment by sbkmlvb** [comment]
![Uploading 微信截图_20221031210225.png…]()
![ccc](https://user-images.githubusercontent.com/54029752/199014626-9737b343-1eb1-4247-b0fa-49d4e56bb07e.png)

每次刷新都会出现这个

**Comment by hengyanqwe** [comment]
我也遇到了，不知道是不是这个问题，https://github.com/steedos/steedos-platform/issues/3731#issuecomment-1772262721

**Comment by hengyanqwe** [comment]
2022年的老问题了，到现在也没好
